bootstrap/11-pm.sh

```diff
@@ -113,5 +113,7 @@ if ${CREATE_ONLY_CHROOT}; then
 
     cleanup
 
+    success "${OS} chroot environment is ready."
```

Comment:
When Pieman finishes building images successfully, it prints
> ${IMAGE}${extension} was built. Use Etcher (https://etcher.io) to burn it to your SD card.

By analogy in this case it should be something like
> ${R} was built. Use BASE_DIR to build images based on the chroot environment.